tests/all-tests.js

```diff
@@ -21,6 +21,28 @@ https://github.com/GPII/universal/blob/master/LICENSE.txt
 var fluid = require("infusion"),
     kettle = fluid.require("kettle");
 
+// TODO: Remove this once we're done profiling.
```

Comment:
Can this TODO block be removed too?

Reply:
Hmm... [See above](https://github.com/GPII/universal/pull/718#discussion_r262555196).    If I remove this block and compare against master, then there is no difference.  It's very likely that it's safe to remove it.

Reply:
Yes, I got rid of both in my own later work.

Reply:
Thanks, @the-t-in-rtf for the confirmation(s).